chain/consensus/delegcns/delegated.go

```
@@ -83,6 +83,7 @@ func NewDelegatedConsensus(
 	genesis chain.Genesis,
 	netName dtypes.NetworkName,
 ) consensus.Consensus {
+	sn, _ := address.SubnetIDFromString(string(netName))
```

Why error is ignored here?